ftree/node.go

```diff
@@ -0,0 +1,59 @@
+package ftree
+
+import (
+	"fmt"
+	"sort"
+)
+
+type Node map[string]Node
+
+// Print recurses through n and its member Nodes to pretty-print a Node as a
+// tree such as:
+//
+//     ├── ftree
+//     │   ├── node.go
+//     │   ├── tree.go
+//     │   └── tree_test.go
+//     └── main.go
+//
+// It should initially be called with as n.Print(0, nil) as the arguments are
+// primarily interesting during recursion.
+func (n Node) Print(indent int, leaders []string) {
+	var keys []string
+	for k := range n {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	for i, part := range keys {
```

Comment:
Why call a single element in `keys` `part` rather than `key`? Should `keys` be `parts`?